data/sql/updates/pending_db_world/rev_1734751988808151700.sql

```diff
@@ -0,0 +1,14 @@
+-- Angered, Suffering & Hungering Soul Fragment respawn
+UPDATE `creature` SET `spawntimesecs` = 15 WHERE guid IN (    
```

Comment:
Missing backticks on guid and id1.
Missing new line at end of file.

Reply:
I am not even done...